torchaudio/datasets/speechcommands.py

```diff
@@ -20,6 +20,42 @@
     "https://storage.googleapis.com/download.tensorflow.org/data/speech_commands_v0.02.tar.gz":
     "6b74f3901214cb2c2934e98196829835",
 }
+LABELS = [
+    "bed",
+    "bird",
+    "cat",
+    "dog",
+    "down",
+    "eight",
+    "five",
+    "follow",
+    "forward",
+    "four",
+    "go",
+    "happy",
+    "house",
+    "learn",
+    "left",
+    "marvin",
+    "nine",
+    "no",
+    "off",
+    "on",
+    "one",
+    "right",
+    "seven",
+    "sheila",
+    "six",
+    "stop",
+    "three",
+    "tree",
+    "two",
+    "up",
+    "visual",
+    "wow",
+    "yes",
+    "zero",
+]
```

Comment:
I gave some thoughts of this, but can you put this in test? Logically, this should belong here but since it's not used in the implementation, it might be confusing later.